liquid-rust-typeck/src/bblock_env.rs

```diff
@@ -11,20 +11,21 @@ use liquid_rust_lrir::{
 };
 
 pub struct BBlockEnv {
-    pub ghost_vars: OrderedHashMap<GhostVar, Ty>,
+    pub ghost_vars: BTreeMap<GhostVar, Ty>,
```

Comment:
We need to preserve insertion order and inserting them on a btree would reorder them. In practice, I think we always use the "alphabetical" order would match the insertion order but maybe that's too much to ask from context.